### PR TITLE
HIPP-987: Produce an API

### PR DESCRIPTION
**Pull request by PaulCDurham**
https://jira.tools.tax.service.gov.uk/browse/HIPP-987

### CODE REVIEW

**Review by victorarbuesmallada — DISMISSED** [review]

File: app/views/components/RelatedContent.scala.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * limitations under the License.
+ *@
+
+@import viewmodels.RelatedContentLink
+
+@this()
+
+@(relatedContentLinks: Seq[RelatedContentLink])(implicit request: RequestHeader, messages: Messages)
+
+    <div class="gem-c-related-navigation">
```
Comment (rdawson-hmrc) [inline app/views/components/RelatedContent.scala.html:23]: the prototype includes various data attributes on this element which look like they are related to google analytics, not sure if they are/will be required might be worth checking with Jono

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -788,3 +788,51 @@ header {
...
+  border-top: 2px solid #1d70b8;
+  margin-bottom: 60px;
+}
+
+.gem-c-related-navigation__main-heading {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 700;
+  font-size: 16px;
```
Comment (rdawson-hmrc) [inline app/assets/stylesheets/application.scss:803]: duplicated font-size, the first one is redundant

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -788,3 +788,51 @@ header {
...
+  font-size: 1rem;
+  line-height: 1.25;
+  margin-top: 15px;
+  margin-bottom: 10px;
+}
+
+@media print {
```
Comment (rdawson-hmrc) [inline app/assets/stylesheets/application.scss:810]: we don't specify print styles for anything else, probably don't need this

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -788,3 +788,51 @@ header {
...
+  .gem-c-related-navigation__main-heading {
+    font-family: sans-serif;
+  }
+}
+@media (min-width: 40.0625em) {
+  .gem-c-related-navigation__main-heading {
+    font-size: 19px;
```
Comment (rdawson-hmrc) [inline app/assets/stylesheets/application.scss:817]: same here, duplicated font-size

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -788,3 +788,51 @@ header {
...
+    font-size: 1.1875rem;
+    line-height: 1.3157894737;
+  }
+}
+@media print {
```
Comment (rdawson-hmrc) [inline app/assets/stylesheets/application.scss:822]: more print styles, can omit this

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -788,3 +788,51 @@ header {
...
+
+.gem-c-related-navigation__link-list {
+  padding: 0;
+  list-style: none;
+  margin-bottom: 1.25em;
+}
+
+.gem-c-related-navigation__link {
+  margin-bottom: 0.8em;
+  font-size: 1rem;
```
Comment (rdawson-hmrc) [inline app/assets/stylesheets/application.scss:837]: we have a real mix of units for fonts and various other things, sometimes px sometimes em sometimes rem - I'll do a tech-ticket to try and add some consistency

File: app/views/components/RelatedContent.scala.html (hunk 1 above)
Comment (PaulCDurham) [reply app/views/components/RelatedContent.scala.html:23]: They aren't required. GA in MDTP uses a common code snippet that comes in via one of the platform libs.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (PaulCDurham) [reply app/assets/stylesheets/application.scss:803]: Done

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (PaulCDurham) [reply app/assets/stylesheets/application.scss:810]: Done

File: app/assets/stylesheets/application.scss (hunk 4 above)
Comment (PaulCDurham) [reply app/assets/stylesheets/application.scss:817]: Done

File: app/assets/stylesheets/application.scss (hunk 5 above)
Comment (PaulCDurham) [reply app/assets/stylesheets/application.scss:822]: Done

File: app/assets/stylesheets/application.scss (hunk 6 above)
Comment (PaulCDurham) [reply app/assets/stylesheets/application.scss:837]: That would be good. Should we produce a Confluence page maybe at the same time with the rules so we know what to do when importing CSS?

**Review by rdawson-hmrc — APPROVED** [review]

**Review by victorarbuesmallada — APPROVED** [review]

**Pull request merged**